package.json

```diff
@@ -477,7 +465,7 @@
               "path": "${command:AskForTDMLPath}"
             },
             "debugServer": 4711,
-            "openHexView": false,
+            "openDataEditor": false,
```

Comment:
Testing on Windows...
"openDataEditor=true" did not work for me.
Had to open DataEditor using command pallet then select file. What happens if I select the wrong file? Will it display the location in this wrong file?
HexView, I think, made a copy of the file and opened that. Do we want to replicate that functionality?

Reply:
That't odd it worked fine for me on Mac.

Reply:
@shanedell @lrbarber 

> That't odd it worked fine for me on Mac.

I'm pretty sure it's a Windows only issue, then. I'm still working on configuring my Windows environment to develop the extension. Once I get the environment set up (which is a PITA for Windows, as usual), I will try to test this and get a working fix in.

Reply:
Currently, I'm unable to actually get anything running because of this error I'm receiving while attempting to run the DFDL Debugger.

![daffodil-debugger_bat_error](https://github.com/apache/daffodil-vscode/assets/30351915/b5cd76a4-2730-49e2-834c-a7079cb287e4)

**Error Message** : 

```bash
PS C:\Users\Orion\Documents\GitHub\PNG> c:\Users\Orion\Documents\GitHub\daffodil-vscode\daffodil-debugger-3.5.0-1.4.0-SNAPSHOT\bin\daffodil-debugger.bat --listenPort 4711 -J"--add-opens java.base/java.lang=ALL-UNNAMED"
Unrecognized option: --add-opens
Error: Could not create the Java Virtual Machine.
Error: A fatal exception has occurred. Program will exit.
```